pkgs/applications/editors/lapce/utils.nix

```
@@ -0,0 +1,80 @@
+{
+  stdenvNoCC,
+  curl,
+  cacert,
+  zstd,
+  linkFarm,
+}:
+let
+  fetchPluginTarballFromRegistry =
+    {
+      author,
+      name,
+      version,
+      hash,
+      meta,
+    }:
+    stdenvNoCC.mkDerivation (
+      let
+        url = "https://plugins.lapce.dev/api/v1/plugins/${author}/${name}/${version}/download";
+        file = "lapce-plugin-${author}-${name}-${version}.tar.zstd";
+      in
+      {
+        name = file;
+        nativeBuildInputs = [
+          curl
+          cacert
+        ];
+        phases = [ "installPhase" ];
```

isn't `phase` depreciated in favor of `dont[...] = true;`?
- https://github.com/NixOS/nixpkgs/issues/28910

```suggestion
      dontUnpack = true;
      dontBuild = true;
```

(same for the second one)